task/blue-green-deploy/0.1/blue-green-deploy.yaml

```diff
@@ -29,7 +29,7 @@ spec:
       description: The newer version of the deployment to be deployed in another zone
   steps:
     - name: change-deployment
-      image: quay.io/openshift/origin-cli:latest
+      image: quay.io/openshift/origin-cli:4.4
```

Comment:
@chmouel  @vdemeester  Using this particular tag and not the digest because with quay if a tag is updated the previous digest becomes invalid ?

So are we fine with using this specific tag ?

Reply:
yes that is fine